bin/eslint-formatters-calypso/src/formatters/eslines.js

```diff
@@ -0,0 +1,13 @@
+const childProcess = require( 'child_process' );
+const linesAndRules = require( './lines-modified-and-specific-rules' );
+const parsingErrors = require( './parsing-errors' );
+
+module.exports = function( report ) {
+	const argsBranchName = [ 'rev-parse', '--abbrev-ref', 'HEAD' ];
+	const branchName = childProcess.spawnSync( 'git', argsBranchName ).stdout.toString().trim();
```

Comment:
Formatters [need](https://github.com/eslint/eslint/blob/master/lib/cli.js#L84) to run synchronously.